src/TraceEvent/TraceLoggingEventId.cs

```diff
@@ -10,19 +11,20 @@ namespace Microsoft.Diagnostics.Tracing
     /// <summary>
     /// TraceLoggingEvnetId is a class that manages assigning event IDs (small 64k numbers)
     /// to TraceLogging Style events (which don't have them).  Because TraceEvent uses EventIDs
-    /// so fundamentally this difficiency is very problematic.   
+    /// so fundamentally this deficiency is very problematic.   
```

Comment:
@northtyphoon It seems to be covered well enough already. I mostly assume that people don't read that, or do a cursory look at best. I don't let it bother me when deviations happen and I'm always willing to help if someone isn't sure how to break things up. :smile: